client/lobby/CSavingScreen.cpp

```diff
@@ -52,7 +50,9 @@ const CMapInfo * CSavingScreen::getMapInfo()
 
 const StartInfo * CSavingScreen::getStartInfo()
 {
-	return localSi;
+	if (localMi)
+		return localMi->scenarioOptionsOfSave;
+	return LOCPLINT->cb->getStartInfo();
```

Comment:
Are local interface always exist? Suggest to test mp scenarios

Reply:
Yes, always exists.
In MP game switches between different LOCPLINT'S.

Old code was also using LOCPLINT unconditionally.
LOCPLINT is null during pregame, but saving screen is not available in it.